doc/nrf/releases_and_maturity/release_notes.rst

```diff
@@ -19,7 +19,7 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
-   releases/release-notes-changelog
+   releases/release-notes-2.9.0-nrf54h20-rc1
```

Comment:
Please add path for the "Migration guide for |NCS| v2.9.0-nRF54H20-rc1" migration guide after the PR https://github.com/nrfconnect/sdk-nrf/pull/19560/files is merged here: nrf\releases_and_maturity\migration_guides.